Added tutorial to phase-field module #5414

itgreenquist opened the pull request:
Added input files for a phase-field tutorial that can be found at the wiki: http://mooseframework.org/wiki/MooseTutorials/IronChromiumDecomposition/

moosebuild commented:
Results of testing f01c25b77db20ffe5ce30d43ffc6657353b36a39 using moose_PR_pre_check recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19832

friedmud commented:
Wait!  Before you close this again!

The current error you're seeing is that your commit message must reference an issue number.  In your case you just need to put `#5414` into your commit message.

If you're not very familiar with git you can do that by just doing a `git commit --amend`... it will let you modify your commit message for your last commit.

You don't need to close it.  Just ammend the commit message and then `git push --force` to push it up to GitHub

friedmud commented:
Let us know if you need more help

friedmud commented:
To repeat: you don't need to close the PR.  Once you modify your commit message and force push... the PR will automatically update and the tests will start again

moosebuild commented:
Results of testing d025e9e081d6d1a2ffc2f1a13e44b1fcf6b9d9cf using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19842

moosebuild commented:
Results of testing d025e9e081d6d1a2ffc2f1a13e44b1fcf6b9d9cf using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19844

moosebuild commented:
Results of testing d025e9e081d6d1a2ffc2f1a13e44b1fcf6b9d9cf using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19843

moosebuild commented:
Results of testing d025e9e081d6d1a2ffc2f1a13e44b1fcf6b9d9cf using moose_PR_app_tests recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19845

itgreenquist commented:
Thanks for your help I think I got it to work.

moosebuild commented:
Results of testing d025e9e081d6d1a2ffc2f1a13e44b1fcf6b9d9cf using moose_PR_modules_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19846

tonkmr commented on modules/phase_field/tutorials/spinodal_decomposition/s1_testmodel.i:
```diff
@@ -0,0 +1,147 @@
+  # periodic BC as is usually done on phase-field models
+  [./Periodic]
+    [./c_bcs]
+      variable = c
```
If you leave the variable line out, it will automatically apply the BC to every variable so you will only need one block.

tonkmr commented on modules/phase_field/tutorials/spinodal_decomposition/s4_mobility.i:
```diff
@@ -0,0 +1,208 @@
+
+[GlobalParams]
+  block = 0           # The generated mesh is used for all materials and kernels
+  enable_jit = true   # Enables Just-in-time compilation for parsed materials
```
I think that just-in-time compilation is enabled by default, so it is actually being run in every one of these input files

moosebuild commented:
Results of testing 908e9c70b1cd6b18b4359220cdcd9d749e722fec using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19898

moosebuild commented:
Results of testing 908e9c70b1cd6b18b4359220cdcd9d749e722fec using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19899

moosebuild commented:
Results of testing 908e9c70b1cd6b18b4359220cdcd9d749e722fec using moose_PR_app_tests recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19901

moosebuild commented:
Results of testing 908e9c70b1cd6b18b4359220cdcd9d749e722fec using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19900

moosebuild commented:
Results of testing 908e9c70b1cd6b18b4359220cdcd9d749e722fec using moose_PR_modules_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19902

Commit:
Update input files to remove unnecessary code #5414

Conflicts:
	modules/phase_field/tutorials/spinodal_decomposition/s1_testmodel.i
	modules/phase_field/tutorials/spinodal_decomposition/s2_fasttest.i
	modules/phase_field/tutorials/spinodal_decomposition/s3_decomp.i
	modules/phase_field/tutorials/spinodal_decomposition/s4_mobility.i
	modules/phase_field/tutorials/spinodal_decomposition/s5_energycurve.i

moosebuild commented:
Results of testing 6cfcbf3c88d7cd36c58f3281f58f9918cad0107b using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19903

moosebuild commented:
Results of testing 6cfcbf3c88d7cd36c58f3281f58f9918cad0107b using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19904

moosebuild commented:
Results of testing 6cfcbf3c88d7cd36c58f3281f58f9918cad0107b using moose_PR_app_tests recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19906

dschwen commented:
Hey Ian, it looks like your branch got messed up, try the following on your feature branch:

```
git fetch upstream
git reset --hard upstream/devel
git cherry-pick 6cfcbf3
git push -f
```

tonkmr commented:
Daniel's instructions will delete all of the extra commits that are showing (250+) and just leave your one commit.

moosebuild commented:
Results of testing 6cfcbf3c88d7cd36c58f3281f58f9918cad0107b using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19905

moosebuild commented:
Results of testing 6cfcbf3c88d7cd36c58f3281f58f9918cad0107b using moose_PR_modules_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19907

itgreenquist commented:
I followed the instructions but the commits are still showing up

itgreenquist commented:
Never mind, just needed to refresh the page

moosebuild commented:
Results of testing 64d72d3db2bc9d01b0c5265e535987143f45790d using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19908

moosebuild commented:
Results of testing 64d72d3db2bc9d01b0c5265e535987143f45790d using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19909

moosebuild commented:
Results of testing 64d72d3db2bc9d01b0c5265e535987143f45790d using moose_PR_app_tests recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19911

moosebuild commented:
Results of testing 64d72d3db2bc9d01b0c5265e535987143f45790d using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19910

moosebuild commented:
Results of testing 64d72d3db2bc9d01b0c5265e535987143f45790d using moose_PR_modules_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19912
